client/my-sites/themes/thanks-modal.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/jsx-no-bind */
```

Comment:
Please don't unless there's a way more compelling reason than elsewhere :-)

What about changing the `trackClick` method as follows instead:

```js
boundTrackClick: function( eventName ) {
  return Helpers.trackClick.bind( null, 'current theme', eventName );
}
```

That should allow usage like

```js
onClick={ this.boundTrackClick( 'features' )}
```